src/mpid/ch3/channels/nemesis/netmod/llc/llc_send.c

```diff
@@ -849,7 +847,8 @@ int llc_poll(int in_blocking_poll, llc_send_f sfnc, llc_recv_f rfnc)
                         dprintf("llc_poll,ref_count=%d,pack_buf=%p\n", req->ref_count,
                                 REQ_FIELD(req, pack_buf));
                         MPL_free(REQ_FIELD(req, pack_buf));
-			MPIR_Segment_free(seg);
+
+                        MPIR_Segment_free(seg);
                     }
```

Comment:
Should here be unchanged ?

Reply:
We are allocating the segment, so it needs to be freed, right?  What's the issue?

Reply:
This change is a simple format clean up, right ?

Reply:
Oh, yes.  Sorry, I was looking at the wrong commit.

Reply:
This line is still shown as changed. Is it a tab -> space format change ?

Reply:
Fixed.